scout/server/blueprints/variants/templates/variants/str-variants.html

```diff
@@ -41,13 +41,14 @@
       <thead>
         <tr>
           <th class="col-xs-1" title="Rank position">Rank</th>
```

Comment:
You are using 11 cols out of 12, you could use the last one to give some more space to one of the existing cols? It's up to you!

Reply:
Ok!